src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-form/pool-form.component.html

```diff
@@ -566,6 +577,11 @@
                      placeholder="e.g., 10GiB"
                      defaultUnit="GiB"
                      cdDimlessBinary>
+              <cd-help-text>
+                <span i18n>Leave it blank or specify 0 to disable this quota.</span>
+                <br>
+                <span i18n>A valid quota should be greater than 0.</span>
+              </cd-help-text>
```

Comment:
Looks nice, just one thing, we  are using same help text for bytes and quota.
Is it by miss ? 

For quota:
Also its seem understood that 0 value will disable quota, means no quota, so perhaps you can rethink , if you want that to mention to user.
Rather we should specifiy what is meant by "max objects" and "max bytes" what will they do if user enters any value there ?

Reply:
No, both apply to the quota. If specified as `0`, it means the quota is disabled. Given that they are part of quotas, it should be self-explanatory what the maximum bytes and maximum objects a user wants to consume.